habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -1110,6 +1110,24 @@ def _eval_checkpoint(
 
                         rgb_frames[i] = []
 
+                    # A gfx-replay list of keyframes for the episode. This is a JSON string that
+                    # should be saved to a file; the file can be read by visualization tools
+                    # (e.g. import into Blender for screenshots and videos).
+                    gfx_replay_uuid = "gfx_replay_keyframes_string"
```

Comment:
I'm open to suggestions for some place to declare this UUID as a global so it can be re-used in `rearrange_task.py`.